src/main/drivers/adc.c

```diff
@@ -96,9 +96,16 @@ bool adcVerifyPin(ioTag_t tag, ADCDevice device)
     }
 
     for (int map = 0 ; map < ADC_TAG_MAP_COUNT ; map++) {
+#if defined(STM32F1)
```

Comment:
Not really sure about adding complexity to code that is used in non - F1 builds. When do you think we should stop doing this for only F1's sake?